src/static/helpers/slasHelper.ts

```diff
@@ -16,6 +16,9 @@ import {
 } from '../../lib/shopperLogin';
 import ResponseError from '../responseError';
 
+const warningSlasSecretMsg =
+  'This function can run on client-side. You are potentially exposing SLAS secret on browser. Make sure to keep it safe and secure!';
+
```

Comment:
I think we should also include a similar warning in the README with the addition of a private SLAS client helper method

Reply:
I left a note in readme and added a section for private slas client